OCPBUGS#43078: RE-added statement about pods and conversion to dual-s… (#83941)
openshift/openshift-docs: modules/nw-dual-stack-convert.adoc

[id="nw-dual-stack-convert_{context}"]
= Converting to a dual-stack cluster network

As a cluster administrator, you can convert your single-stack cluster network to a dual-stack cluster network. 

[IMPORTANT]
====

> After converting your cluster to use dual-stack networking, only newly created pods are assigned IPv6 addresses. For any pods created before the conversion, you must re-create the pods for them to receive IPv6 addresses.

Reviewer: Just a nitpick suggestion:
```suggestion
After converting your cluster to use dual-stack networking, only new pods are assigned IPv6 addresses. For any pods created before the conversion, you must re-create the pods for them to receive IPv6 addresses.
```

Author: Thanks. Updated!

Reviewer: Or you could take it even further, something in this direction:
```suggestion
After converting your cluster to use dual-stack networking, 
you must re-create any existing pods that you wish to receive IPv6 addresses 
because only new pods are assigned IPv6 addresses.
```

Author: Updated.

[id="nw-dual-stack-convert_{context}"]
= Converting to a dual-stack cluster network

As a cluster administrator, you can convert your single-stack cluster network to a dual-stack cluster network. 

After converting to a IPv4/IPv6 dual-stack network, new and existing pods have IPv4/IPv6 dual-stack networking enabled. If you convert from a single-stack cluster network to a IPv6/IPv4 dual-stack cluster, only new pods have IPv6/IPv4 dual-stack networking enabled

> 

Reviewer 2: I suggest rollback the original to 

```
After converting to dual-stack networking only newly created pods are assigned IPv6 addresses. Any pods created before the conversion must be recreated to receive an IPv6 address.
```
from https://github.com/openshift/openshift-docs/pull/75232/files

Author: Thanks, @zhaozhanqi . Updated but with some minor grammar changes.
